data/json/monsters/mammal.json

```diff
@@ -1759,6 +1759,83 @@
     },
     "flags": [ "ANIMAL", "DOGFOOD", "CANPLAY", "HEARS", "KEENNOSE", "PATH_AVOID_DANGER_1", "SEES", "SMELLS", "WARM" ]
   },
+  {
+    "id": "mon_dog_gswiss",
+    "copy-from": "mon_dog",
+    "type": "MONSTER",
+    "name": { "str_sp": "Greater Swiss" },
+    "description": "The biggest of four breeds of Swiss mountain dogs.  The Greater Swiss mountain dog was originally meant to pull carts, herd cattle and guard the homestead.  This one seems feral, but perhaps you could try taming it?",
+    "volume": "55000 ml",
+    "weight": "55 kg",
+    "hp": 48,
+    "speed": 130,
+    "aggression": 4,
+    "morale": 8,
+    "melee_skill": 3,
+    "melee_dice_sides": 2,
+    "melee_damage": [ { "damage_type": "cut", "amount": 6 } ],
+    "vision_night": 6,
+    "harvest": "dog_with_skull_leather",
+    "special_attacks": [ [ "LUNGE", 5 ] ],
+    "reproduction": { "baby_monster": "mon_dog_gswiss_pup", "baby_count": 7, "baby_timer": 300 },
+    "//2": "1-7 puppies & 300-320 days per-litter for size medium canines",
+    "upgrades": { "half_life": 90, "into": "mon_bigdog_greatestswiss" },
```

Comment:
Half-life like this is not the right way to mutate non-zomb critters. Use either a group with something like 75% normal and unmutating and 25% mutant or use spawn timers to increase their spawn rate over time without spawned ones mutating. 
You can also have them have a low chance of giving birth to mutant puppies.